dp-collection-api.nomad

```diff
@@ -0,0 +1,76 @@
+job "dp-collection-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "publishing" {
+    count = "{{PUBLISHING_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "publishing"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-collection-api-publishing" {
+      driver = "docker"
+
+      artifact {
+        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-collection-api/{{PROFILE}}/{{RELEASE}}.tar.gz"
+      }
+
+      config {
+        command = "${NOMAD_TASK_DIR}/start-task"
+
+        args = ["./dp-collection-api"]
+
+        image = "{{ECR_URL}}:concourse-{{REVISION}}"
+      }
+
+      service {
+        name = "dp-collection-api"
+        port = "http"
+        tags = ["publishing"]
+
+        check {
+          type     = "http"
+          path     = "/health"
+          interval = "10s"
+          timeout  = "2s"
+        }
+      }
+
+      resources {
+        cpu    = "{{PUBLISHING_RESOURCE_CPU}}"
+        memory = "{{PUBLISHING_RESOURCE_MEM}}"
+
+        network {
+          port "http" {}
+        }
+      }
+
+      template {
+        source      = "${NOMAD_TASK_DIR}/vars-template"
+        destination = "${NOMAD_TASK_DIR}/vars"
+      }
+
+      vault {
+        policies = ["dp-collection-api-publishing"]
```

Comment:
as this app is run just in publishing the policies name should just be `dp-collection-api`. Only if the app runs in both we append web/publishing to the name.

Reply:
we've decide to keep this because it is unclear whether we'll need the app to run in `web` as well

Reply:
Currently we have just the publishing group, and the web group has already been deleted, I would personally stick same style we use for other app's. If we add web, then we can always update it. But, I don't mind if the decision is made consciously. I am happy for the PR to move forward

Reply:
Yeah, the decision was definitely made conciously 🙂